frontend/appflowy_flutter/lib/plugins/database/application/cell/bloc/relation_row_search_bloc.dart

```diff
@@ -24,11 +26,22 @@ class RelationRowSearchBloc
       (event, emit) {
         event.when(
           didUpdateRowList: (List<RelatedRowDataPB> rowList) {
-            allRows.clear();
-            allRows.addAll(rowList);
-            emit(state.copyWith(filteredRows: allRows));
+            allRows
```

Comment:
Does it make sense to move the `allRows` field to the `RelationRowSearchState` class?